brewtils/specification.py

```diff
@@ -96,6 +96,18 @@ def _is_json_dict(s):
         "description": "Refresh token for authentication",
         "required": False,
     },
+    "http_proxy": {
```

Comment:
Does it makes sense that these are two separate settings?  If I set `https_proxy`, but then `ssl_enabled` is set to False, then presumably my proxy setting just gets ignored, and vice versa.  Is there ever a situation in which setting both of these is a meaningful thing to do? If not, then maybe this ought to just be a single setting that is provided in the style of "<hostname>:<port>", with the http vs https part being determined by that `ssl_enabled` setting.